data_type: "bin"
stop_each_frame: false
pub_ground_truth: true
pub_frequency: 10.0
data_path: "/home/awesome/Documents/Datasets/SemanticKitti/dataset/sequences/00"
start_frame: 0      # kitti: 00:4390 01:150 02:860 05:2350 07:630, doals: 0
end_frame: 10000        # kitti: 00:4530 01:250 02:950 05:2670 07:820, doals: 250
min_range: 2.7
max_range: 80.0
min_height: -1.5
max_height: 3.0
delta_range: 0.5       # kitti: 00:4390 01:150 02:860 05:2350 07:630, doals: 0
delta_azimuth: 0.5
delta_polar: 3.0
res_range: 0.25
res_azimuth: 1.5
res_height: 0.25
downsample: false
voxel_leaf: 0.1
window_size: 4
cache_size: 4         # kitti: 00:4 
sim_thres: 0.7       # kitti: 00:0.75